src/content/docs/style-guide/ui-writing/capitalization.mdx

```diff
@@ -42,6 +62,8 @@ Capitalization helps communicate the importance of specific words and can be use
 | Abbreviated numbers above 999 | 1K, 1M, 1B     |
 | Acronyms                      | API, CDN, HTTP |
 | Time zones                    | PDT, CST, BCN  |
+| Time periods                 |AM or PM |https://docs.newrelic.com/docs/style-guide/ui-writing/numbers-dates-and-times#date-and-time
```

Comment:
@mmfred was this supposed to be a link like this: [AM or PM](/docs/style-guide/ui-writing/numbers-dates-and-times#date-and-time)?